api/v1alpha1/volumesnapshotrestore_types.go

```diff
@@ -44,6 +45,18 @@ type DMBRef struct {
 	ResticRepository string `json:"resticrepository,omitempty"`
 }
 
+type VolumeSnapshotRestorePhase string
+
+const (
+	DatamoverRestoreVolSyncPhaseCompleted VolumeSnapshotRestorePhase = "VolSyncCompleted"
```

Comment:
Lets change this to `SnapshotBackupCompleted` and  like `SnapshotRestoreCompleted` instead of using `Volsync` as a prefix in phase. WDYT ? 
Alternatives:
- `SnapshotBackupDone` / `SnapshotRestoreDone` maybe `XYZDone` differentiates this from `Completed` more appropriately